specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/examples/OperationsList.json

```diff
@@ -5,7 +5,7 @@
   "responses": {
     "200": {
       "body": {
-        "nextLink": "https://management.azure.com/subscriptions/subscriptionId/providers/Microsoft.ResourceGraph/?api-version=2019-09-01-preview&$top=10&$skiptoken=skiptoken",
```

Comment:
Is the actual response a "nextLink" key with null value, or should this line be omitted entirely?

Reply:
Adding @yangmsft

Reply:
If we can omit it and validation allows it, let's omit the line.